application/microservices/entando-hub-catalog-ms/src/main/java/com/entando/hub/catalog/service/CatalogService.java

```diff
@@ -22,10 +22,15 @@ public CatalogService(CatalogRepository catalogRepository, OrganisationService o
         this.catalogRepository = catalogRepository;
         this.organisationService = organisationService;
     }
-    public List<Catalog> getCatalogs() {
-        return catalogRepository.findAll();
+    public List<Catalog> getCatalogs(String username, boolean userIsAdmin) {
+        if (userIsAdmin) {
```

Comment:
We can reatrieve the isAdmin value using the helper and not passing the parameter if we do the change in my other comments